backend/tests/services/rag/test_text_sanitizer.py

```
@@ -0,0 +1,139 @@
+import pytest
+from llama_index.core import Document
+from llama_index.core.schema import TextNode
+
+from app.services.rag.index.indexer import DocumentIndexer
+from app.services.rag.preprocess.text_sanitizer import sanitize_text_for_indexing
+from app.services.rag.storage.base import BaseStorageBackend
+from app.services.rag.storage.chunk_metadata import ChunkMetadata
+
+
+class _FakeStorageBackend(BaseStorageBackend):
+    SUPPORTED_RETRIEVAL_METHODS = ["vector"]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)